sources/packages/web/src/components/aest/institution/modals/AssessOfferingChangeModal.vue

```diff
@@ -19,11 +19,18 @@
       </div>
     </template>
     <template v-slot:footer>
-      <footer-buttons
-        :primaryLabel="submitLabel"
-        @primaryClick="submitForm"
-        @secondaryClick="dialogClosed"
-      />
+      <check-a-e-s-t-permission-role
+        :role="Role.InstitutionApproveDeclineOfferingChanges"
+      >
+        <template v-slot="{ isReadonly }">
+          <footer-buttons
+            :primaryLabel="submitLabel"
+            @primaryClick="submitForm"
+            @secondaryClick="dialogClosed"
+            :disablePrimaryButton="isReadonly"
+          />
+        </template>
+      </check-a-e-s-t-permission-role>
```

Comment:
Same as previous comment. this modal should't show up as the button to open this modal will be disabled.

Reply:
same as above comment